src/main/java/com/metamx/emitter/service/ServiceMetricEvent.java

```diff
@@ -38,20 +38,23 @@ public static Builder builder()
   private final DateTime createdTime;
   private final ImmutableMap<String, String> serviceDims;
   private final Map<String, Object> userDims;
+  private final String feed;
   private final String metric;
   private final Number value;
 
   private ServiceMetricEvent(
       DateTime createdTime,
       ImmutableMap<String, String> serviceDims,
       Map<String, Object> userDims,
+      String feed,
```

Comment:
Please include a constructor that is signature compatible with the old one. Ideally we should be moving to a builder but that is probably outside the scope of this PR